src/panel/servicePanel.jsx

```diff
@@ -0,0 +1,64 @@
+/* global _ */
+import React from 'react';
+import Panel from 'src/components/ui/Panel';
+import CategoryService from '../adapters/category_service';
+import nconf from '@qwant/nconf-getter';
+
+class ServicePanel extends React.Component {
+  render() {
+    return <Panel
+      resizable
+      title={_('Qwant Maps services', 'service panel')}
+      minimizedTitle={_('Show Qwant Maps services', 'service panel')}
+      className={'service_panel'}
+    >
+      <div className="service_panel__categories">
+        {
+          CategoryService.getCategories().map(item =>
+            <button className="service_panel__category" type="button" key={item.name}
+              onClick={() => {
+                window.app.navigateTo(`/places/?type=${item.name}`);
+              }}>
```

Comment:
When https://github.com/QwantResearch/erdapfel/pull/404 is merged you'll be able to write these callbacks inline if you prefer.

Reply:
Done! If you prefer to inline this king of call (there are 3) for readability, you can :)